src/scripts/populate-dev.js

```diff
@@ -1,10 +1,20 @@
 import FS from'q-io/fs';
-import cconsole  from 'colored-console';
+import Winston  from 'winston';
```

Comment:
Is there a reason for not importing `import { logger } from '../helper/logger.js;'`?